test/TestProjects/Lro-Basic-TypeSpec/src/Generated/tspCodeModel.json

```diff
@@ -91,7 +91,7 @@
      "$id": "13",
      "Name": "createProject",
      "ResourceName": "Project",
-     "Description": "The most basic operation.",
+     "Description": "DEPRECATED: Use LongRunningResourceCreateWithServiceProvidedName from the ResourceOperations interface.\nLong-running operation signature to create a resource with a service-provided name.",
```

Comment:
why this changes? Should we update the tsp file of this test project as well?

Reply:
NO. we does not update the tsp file. The description of this operation template is changed in typespec-azure.

Reply:
The long-term fix is to add docs to the test spec, tracked by #3577.  I should be able to do this in the next few days, otherwise someone else can feel free to do it before me.